users/tests/test_api_registration.py

```diff
@@ -37,14 +31,14 @@ def test_registration_form_valid_data(
 
 
 @pytest.mark.django_db()
-def test_registration_form_missing_data(client: Client, user_model: type[User], register_url: str) -> None:
+def test_registration_form_missing_data(client: Client, user_model: type[User], api_register_url: str) -> None:
```

Comment:
Same as above. Please revert and create specific tests for API, do not break registration tests.

Reply:
Misleading test names, please change.